### PR TITLE
Issue #391: Add `priorsense` to FAQ

### LINKED ISSUE #391: Add `priorsense` to FAQ
As described in #182.

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR closes #391.

Self-explanatory / short PR.

If a package is mentioned in FAQ does it need to be in Suggests?

## Checklist

- [x] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

File: vignettes/faq.Rmd [hunk 1]
```diff
@@ -158,6 +158,16 @@ quantile(pred$mean, c(0.01, 0.1, 0.25, 0.5, 0.75, 0.9, 0.99))
 quantile(pred$sd, c(0.01, 0.1, 0.25, 0.5, 0.75, 0.9, 0.99))
 ```
 
+# How can I assess how sensitive the fitted posterior distribution is to the prior distribution used?
+
+We recommend use of the `priorsense` R package [@kallioinen2024detecting] to check how sensitive the posterior distribution is to perturbations of the prior distribution and likelihood using power-scaling analysis:
```
Comment (jamesmbaazam) [inline vignettes/faq.Rmd:163]: Just a drive-by insertion.

```suggestion
We recommend use of the [`priorsense`](https://github.com/n-kall/priorsense) R package [@kallioinen2024detecting] to check how sensitive the posterior distribution is to perturbations of the prior distribution and likelihood using power-scaling analysis:
```

**Review by seabbs — APPROVED** [review]
Nice. I committed James suggestion to include an explicit link to the GitHub for the package.
